packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -6357,8 +6361,10 @@ export interface IScrollablePaneProps extends React.HTMLAttributes<HTMLElement |
     className?: string;
     componentRef?: IRefObject<IScrollablePane>;
     initialScrollPosition?: number;
-    // (undocumented)
+    optimizeForPerformace?: boolean;

```

Comment:
this is a weird prop, and there is a typo in the public api name...

Can we remove this?

Reply:
"performace" should be spelled "performance" just to be clear

Reply:
Earlier, we had [ScrollablePane.types.ts](https://github.com/OfficeDev/office-ui-fabric-react/pull/9527/files?file-filters%5B%5D=.snap&file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx#diff-15d8bc64c906c520722a5be7673dfdd0)
```
export interface IScrollablePaneProps {
  ...
  /**
   * Determines the behavior of Sticky component(s) 
   * having stickyPosition StickyPosition.Header
   */
  stickyAboveContainerBehavior?: IStickyContainerBehavior;

   /**
   * Determines the behavior of Sticky component(s)
   *  having stickyPosition StickyPosition.Footer
   */
  stickyBelowContainerBehavior?: IStickyContainerBehavior;

}

```
```
export interface IStickyContainerBehavior {
  /**
   * If true, it replicates actual element instead of keeping placeholder 
   * for the component which is to be sticky'ed.
   * Calculating placeholder height & width could be an expensive operation.
   * It's a trade off- cost of replicating the element vs. cost of calculating 
    * placeholder height & width.
   */
  notUsePlaceHolder: boolean;

   /**
   * If true, arranges Sticky component(s) based on 
   * Sticky's 'order' prop in ascending order.
   */
  arrangeStickiesBasedOnOrder: boolean;

   /**
   * Determies when sticky behavior kicks in for Sticky component(s).
   * There are some calculations which determine 
   * if a Sticky component is sticky or non-sticky.
   * These calculations can be expensive and affect page load time.
   */
  containerBehavior: StickyContainerBehaviorType;
}
```
```
export enum StickyContainerBehaviorType {
  /**
   * This is the default behavior. It can affect page load time.
   */
  Default = 0,

   /**
   * Sticky component(s) will become sticky or non-sticky based on scrolling.
   * The calculation which determine if a Sticky component is sticky or non-sticky,
   * are done after user interaction (scrolling) and don't affect page load time.
   */
  StickyOnScroll = 1,

   /**
   * Sticky component(s) will always be sticky independent of scrolling.
   * There are no calculations done as the component(s) would always be sticky.
   */
  StickyAlways = 2
}
```
As there were way too many props and various combinations possible for these props, we decided to simplify things and removed `notUsePlaceHolder: boolean;`, 
`arrangeStickiesBasedOnOrder: boolean;`  for stickyContainers, and introduced a single prop - `optimizeForPerformance?: boolean;` . If `optimizeForPerformance` is true, placeholder is not used and stickies are sorted/arranged based on `order` prop.